package.json

```diff
@@ -30,23 +30,23 @@
   "author": "Phillip Cloud <417981+cpcloud@users.noreply.github.com>",
   "license": "Apache-2.0",
   "dependencies": {
-    "@actions/core": "1.11.0"
+    "@actions/core": "1.11.1"
   },
   "devDependencies": {
     "@semantic-release/changelog": "6.0.3",
     "@semantic-release/commit-analyzer": "13.0.0",
     "@semantic-release/git": "10.0.1",
     "@semantic-release/release-notes-generator": "14.0.1",
     "@types/eslint": "9.6.1",
-    "@types/node": "20.16.10",
-    "@typescript-eslint/eslint-plugin": "7.18.0",
-    "@typescript-eslint/parser": "7.18.0",
+    "@types/node": "22.9.0",
+    "@typescript-eslint/eslint-plugin": "8.14.0",
```

Comment:
I think I was leaving this as is because I can't be bothered to deal with the missing rule failures.

Reply:
I guess I **can** be bothered 😂